### PR TITLE
KFSPTS-29608: Fix IWantDocument Requestor lookup to properly load the new address information when the Requestor is changed by the lookup.

### PR DESCRIPTION
**Pull request by nkimble**

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java [hunk 1]
```diff
@@ -1035,6 +1028,26 @@ public ActionForward refresh(ActionMapping mapping, ActionForm form, HttpServlet
         return actionForward;
     }
 
+    private String obtainDeliverToAddressForRefresh(IWantDocument iWantDocument) {
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java:1031]: Two thoughts on this:
1) These functions could be combined with something like findCampusAddressFromNetId(String netId)

2) It looks like this approach will over write a manually edited address with the campus address of the person.  Is that the desired outcome?

**Review by jhulslander — COMMENTED** [review]
I had a couple of thoughts

File: src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java (hunk 1 above)
Comment (nkimble) [reply src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java:1031]: Regarding your two points:

1. I have combined the two functions into one parameterized function as requested.
 
2. Per today's standup discussion with the functional user, the behavior of overwriting any manually entered address is the desired outcome.

**Review by nkimble — COMMENTED** [review]
This is ready for review again.

**Review by jhulslander — APPROVED** [review]
changes look good, I shall merge

**Pull request merged**